values.yaml

```diff
@@ -19,7 +19,7 @@ image: concourse/concourse
 ##      of `concourse/concourse`.
 ## Ref: https://hub.docker.com/r/concourse/concourse/tags/
 ##
-imageTag: "5.4.1"
+imageTag: "5.5.3"
```

Comment:
this might look odd, but it's because we forgot to bump in the original PR that brought 5.5.0 to life in `helm/charts`.